src/paymentservice/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# Payment service configuration file
+
+server.servlet.context-path=/paymentservice
+server.port=8081
+
+# Hosted Payment url should be placed in configMap.
+bambora.hostedpaymentendpoint=https://web.na.bambora.com/scripts/payment/payment.asp
+
+# These values are secret. Do not commit actual values!
+# DEV and TEST to use same merchant and hashkey account. Prod to use values supplied by ISB. 
+bambora.merchantid=changeme
+bambora.hashkey=2D99E892286b428D9D5656E07d1544D9
+
+CRC_ENDPOINT_APPROVED=http://myendpoint/approved
```

Comment:
can we use the ${} placeholder, of change the docker file, otherwise we can override the value, we need to delete the yml config too.

Reply:
Problem is then where does the value contained within the ${} come from when running in an IDE? Could it be set as a Windows enviro variable possibly? This still needs to be worked out.

Reply:
this is going to break our docker though